### DISCUSSION TITLE
Cucumber for Javascript

### DISCUSSION THREAD

**Discussion by illiazo** [allure-framework/allure2#1838] [General Discussion]
Hi, 
I've been trying to run allure on cucumber tests results but I keep getting a NaN page. The Tests are written in Gherkin/Javascript

**Comment by cheshi-mantu** [comment]
@illiazo could you please elaborate a bit?
To recommend something we need to understand, what are you actually doing to get the results.

cheers

**Reply by illiazo** [reply]
@cheshi-mantu we are working with cucumber but with javascript and Gherkin. Our test results are generated in a json file but I believe Allure works with a .xml file, is there any way to convert it into a json`file?

**Comment by cheshi-mantu** [comment]
@illiazo 
Allure TestOps works with the files you are using to generate allure report, i.e. the content of allure-results is to be sent to Allure TestOps, i.e. `*-result.json` files and **all** the complimentary files from the same folders.

**Reply by illiazo** [reply]
@cheshi-mantu but I still don't understand why it doesn't generate any result data (in the dashboard) for our cucumber tests.. do you have any suggestins on how to troubleshoot that?

**Reply by cheshi-mantu** [reply]
have you checked this?
https://github.com/allure-framework/allure-js
